modules/security-context-constraints-about.adoc

```diff
@@ -5,58 +5,86 @@
 = About security context constraints
 
-Similar to the way that RBAC resources control user access, administrators can
-use _security context constraints_ (SCCs) to control permissions for pods. These
-permissions include actions that a _pod_, a collection of containers, can
-perform and what resources it can access. You can use SCCs to define a set of
-conditions that a pod must run with to be accepted into the system.
-
-ifdef::openshift-origin,openshift-enterprise,openshift-dedicated[]
-SCCs allow an administrator to control:
-
-- Whether a pod can run privileged containers.
-- The capabilities that a container can request.
-- The use of host directories as volumes.
-- The SELinux context of the container.
-- The container user ID.
-- The use of host namespaces and networking.
-- The allocation of an `FSGroup` that owns the pod's volumes.
-- The configuration of allowable supplemental groups.
-- Whether a container requires the use of a read only root file system.
-- The usage of volume types.
-- The configuration of allowable `seccomp` profiles.
-
-////
-If you have `cluster-admin` privileges, you can adjust the default SCC policies to
-be more permissive.
-////
-
-The cluster contains nine default SCCs:
-
-* `anyuid`
-* `hostaccess`
-* `hostmount-anyuid`
-* `hostnetwork`
-+
-[WARNING]
-====
-If additional workloads are run on control plane hosts (also known as the master hosts), use caution when providing access to `hostnetwork`. A workload that runs `hostnetwork` on a control plane host is effectively root on the cluster and must be trusted accordingly.
-====
-* `node-exporter`
-* `nonroot`
-* `privileged`
-* `restricted`
-* `pipelines-scc`
+Similar to the way that RBAC resources control user access, administrators can use _security context constraints_ (SCCs) to control permissions for pods. These permissions include actions that a pod can perform and what resources it can access. You can use SCCs to define a set of conditions that a pod must run with to be accepted into the system.
+
+Security context constraints allow an administrator to control:
 
+* Whether a pod can run privileged containers
+* The capabilities that a container can request
+* The use of host directories as volumes
+* The SELinux context of the container
+* The container user ID
+* The use of host namespaces and networking
+* The allocation of an `FSGroup` that owns the pod volumes
+* The configuration of allowable supplemental groups
+* Whether a container requires the use of a read only root file system
+* The usage of volume types
+* The configuration of allowable `seccomp` profiles
+
+[id="default-sccs_{context}"]
+== Default security context constraints
+
+The cluster contains several default security context constraints (SCCs) as described in the table below. Additional SCCs might be installed when you install Operators or other components to {product-title}. By default, all pods and containers use the `restricted` SCC.
```

Comment:
@vdemeester @nikhil-thomas I removed the `pipelines-scc` SCC from the list of default SCCs that ship with OCP, because I believe that it is only created if you install Pipelines. I've added a quick sentence here about that additional SCCs might be installed as you install other operators, etc.

Can you please confirm that this is okay?

Reply:
yes that's right 👍🏼

Reply:
Thanks @stlaz, I'm not sure where I came up with that :confused: